src/headset/fake.c

```diff
@@ -0,0 +1,493 @@
+/* vim: set ts=2 sts=2 sw=2: */
```

Comment:
Can you remove these comments?  There's an `.editorconfig` already for this sort of thing.  Looks like they're here and in `headset.c`, `headset.h`, and `openvr.c`.